docs/releasenotes/3.1.0.rst

```diff
@@ -0,0 +1,56 @@
+:orphan:
+
+=============
+Robocop 3.1.0
+=============
+
+<Release description>
+
+You can install the latest available version by running
+
+::
+
+    pip install --upgrade robotframework-robocop
+
+or to install exactly this version
+
+::
+
+    pip install robotframework-robocop==3.1.0
+
+.. contents::
+   :depth: 2
+   :local:
+
+Most important changes
+======================
+
+Change title (#0)
+-----------------------------------------------
+
+Description
+
+Rule changes
+============
+
+Rules W0310 and W0309 now ignore nested variables (#678)
+------------------------------------------------------------------------------------------
+
+Rules W0310 ``non-local-variables-should-be-uppercase`` and W0309 ``section-variable-not-uppercase``
```

Comment:
Does this statement is true after recent changes to  ``section-variable-not-uppercase`` rule? Ie we don't ignore nested variables in variable name, just in the variable value:

```
*** Variables ***
${VAR_${VAR2}}    ${VAR_${var2}}  # should pass
```

Reply:
Thanks for noticing, updated.